src/client/Client.js

```diff
@@ -338,6 +339,18 @@ class Client extends BaseClient {
       .then(app => new ClientApplication(this, app));
   }
 
+  /**
+   * Tries to fetch the specified guild, if the provided guild isn't public, returns null
```

Comment:
this description needs to be changed

Reply:
I get that the `null` part should be removed from the description from there; but what's the description needed to provided? `Tries to fetch the specified guild, if available to the 'preview' endpoint`, perhaps?